src/styles.scss

```diff
@@ -9,6 +9,53 @@ body {
 body {
   margin: 0;
 }
+.header-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.icon-wrapper {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  width: 40px;
+  height: 40px;
+  margin-right: 20px;
+}
+
+.main-icon {
+  position: absolute;
+  font-size: 40px;
+  margin-right: 8px;
+}
+
+.small-icon-wrapper {
+  position: absolute;
+  width: 15px;
+  height: 15px;
+}
+
+.small-icon {
+  font-size: 15px;
+  z-index: 0;
+}
+
+.title-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 100%;
+}
+
+.check-green {
+  color: green;
+}
+
+.remove-red {
+  color: red;
+}
```

Comment:
regarding class names, as the ui code base is shared for both the connector ui and the broker ui, we should make sure that styles for the broker are prefixed `broker-`

We use https://tailwindcss.com/ , most of these styles might be replaceable.

For example, we could use for header-content: `class="flex items-center justify-between w-full"`

For simple stylings, such as layouting, we should always check if it is better to use tailwind.